pytheus/exposition.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+from pytheus.registry import REGISTRY, Collector, Registry
+
+LINE_SEPARATOR = "\n"
+LABEL_SEPARATOR = ","
+PROMETHEUS_CONTENT_TYPE = "text/plain; version=0.0.4; charset=utf-8"
+
+
+def generate_metrics(registry: Registry = REGISTRY) -> str:
+    """
+    Returns the metrics from the registry in prometheus text format
+    """
+    output = [
```

Comment:
would it make sense to make it a generator to avoid creating the list ?
Not sure if better but it could be chained to add the empty part `""` so we avoid having a list at all 🤔

Reply:
are you suggesting this:

```python
from itertools import chain
output = (
    generate_from_collector(collector, registry.prefix)
    for collector in registry.collect()
)
output =  chain(output, "")
"\n".join(output)
```

Reply:
yeah in theory seems more efficient but I will leave it up to you!
Actually even ignore the chain, just add the final `\n` at the end after building the string maybe 🤔